analyses/infercnv-consensus-cell-type/project-workflows/run-SCPCP000015.sh

```diff
@@ -106,3 +109,47 @@ for sample_id in $sample_ids; do
             output_file = '${html_name}')"
     done
 done
+
+# Next, we run a set of control inferences where the reference is taken from the same library itself
+# Libraries and references to run were determined here: https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/1125
+for sample_id in SCPCS000490 SCPCS000492 SCPCS000750; do
```

Comment:
An alternative to this would be to add an `if` statement within the other `for` loop that checks if any of the sample IDs are one of these IDs. If that's the case then run the script with the internal reference and generate a new notebook.

Reply:
Huh, TIL there is no real quick way in bash to check if a value is in an array. So, while I don't disagree with this in principle, implementing it won't be as pretty as it sounds it seems.

Reply:
Huh? Why would you need to check if it's in an array? You could just do:

```
if [[ $sample_id == "SCPCS000490" || $sample_id == "SCPCS000492" || $sample_id == "SCPCS000750" ]]
```

Reply:
Ah, I was only thinking of defining an array with those sample ids!! This is reasonable/small enough to use or's though!